plugins/processors/topk/topk.go

```diff
@@ -276,7 +276,7 @@ func (t *TopK) push() []telegraf.Metric {
 	}
 
 	// The return value that will hold the returned metrics
-	var ret []telegraf.Metric = make([]telegraf.Metric, 0, 0)
+	var ret = make([]telegraf.Metric, 0, 0)
```

Comment:
Hmm isn't this equivalent to `var ret []telegraf.Metric` which I would prefer then...

Reply:
Then `ret` will be `nil` initially but it is ok for this case. Changed.